### DISCUSSION TITLE
MUV dataset example

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#8829] [General]
Is there any code related to the MUV dataset?

**Comment by rusty1s** [comment]
No. Do you mind sharing a few pointers to the MUV dataset?

**Reply by Chrisable** [reply]
Thank you very much for your reply. I don't know much about the MUV dataset. In fact, what I really want to ask is whether GNN+ [molecular property prediction](https://colab.research.google.com/drive/1I8a0DfQ3fI7Njc62__mVXUlcAleUclnb?usp=sharing#scrollTo=N-FO5xL3mw98) is generally binary classification, and is there a case where GNN+ chemistry is multiclassified. I'm looking for a dataset like GNN+Chemistry that has a specific scenario and is multiclassified.
Looking forward to hearing from you！

**Reply by rusty1s** [reply]
I think in most cases it is indeed binary classification/regression. I know too little sadly on chemistry to be able to point you to a multi-class classification dataset on molecules :(

**Reply by Chrisable** [reply]
In addition to the node classification datasets such as cora, pubmed, and citeseer, does GNN have multi-classification datasets for other scenarios?

**Reply by Chrisable** [reply]
Looking forward to hearing from you

**Reply by rusty1s** [reply]
Yes, most datasets in torch_geometric.datasets are multi-class classification.

**Reply by Chrisable** [reply]
Could you give me a specific scenario? I've seen scenarios for text classification before, such as the R8 dataset, but I found that his label distribution is unbalanced and doesn't work for my training. Are there any other multi-category scenarios for the scene?

**Reply by rusty1s** [reply]
You can browse https://pytorch-geometric.readthedocs.io/en/latest/cheatsheet/data_cheatsheet.html and look for datasets with #classes > 2.
